example/mirror-sfu/README.md

```diff
@@ -0,0 +1,32 @@
+# Simple

```

Comment:
Please update the README to reflect the purpose of the example, it is not clear what this example does, and the README is from a different example.